sklearn/naive_bayes.py

```diff
@@ -703,7 +708,14 @@ class MultinomialNB(_BaseDiscreteNB):
     ----------
     alpha : float, default=1.0
         Additive (Laplace/Lidstone) smoothing parameter
-        (0 for no smoothing).
+        (set alpha=0 and force_alpha=True, for no smoothing).
+
+    force_alpha : bool, default=False
+        If False and alpha is too close to 0, it will set alpha to _ALPHA_MIN.
+        If True, warn user about potential numeric errors
+        and proceed with alpha unchanged.
```

Comment:
For consistency:
```suggestion
    force_alpha : bool, default=False
        If False and alpha is too close to 0, it will set alpha to
        `_ALPHA_MIN`. If True, warn user about potential numeric errors
        and proceed with alpha unchanged.
```